code/pagetypes/Forum.php

```diff
@@ -357,25 +357,51 @@ function getLatestPost() {
 	 * @return int Returns the number of topics (threads)
 	 */
 	function getNumTopics() {
-		return DB::query(sprintf('SELECT COUNT("ID") FROM "ForumThread" WHERE "ForumID" = \'%s\'', $this->ID))->value();
+		return (int)DB::query(
+			sprintf(
+				'SELECT COUNT(Thread.ID)
+				FROM (SELECT Post.ID FROM Post
```

Comment:
Sure, this was simply taken from the similar method in Forum class. Main issue is that Forum and ForumThread are only aware of each other by way of the Post class. I could INNER JOIN both Member and ForumThread on to Post though right @tractorcow ?